hypervisor/arch/x86/trusty.c

```diff
@@ -425,14 +426,20 @@ bool initialize_trusty(struct vcpu *vcpu, uint64_t param)
 		return false;
 	}
 
+	/* FIXME: Temporarily comments out the size check since need to extend
+	 *        the interface, need to add back after the integration done.
+	 */
+	/*
 	if (sizeof(struct trusty_boot_param) !=
 			boot_param.size_of_this_struct) {
 		pr_err("%s: sizeof(struct trusty_boot_param) mismatch!\n",
 			__func__);
 		return false;
 	}
+	*/
```

Comment:
Why leave deadcode here?

Reply:
We need to add back these code after OSloader update the version to 2. Currently, both version 1 and 2 need to be supported, otherwise, the boot will fail.

Please check the Comments before these codes:
+	/* FIXME: Temporarily comments out the size check since need to extend
+	 *        the interface, need to add back after the integration done.
+	 */
+	/*